.circleci/config.yml

```diff
@@ -7,15 +7,10 @@ jobs:
       docker_layer_caching: true
     steps:
       - checkout
-      - run:
-          name: Docker Hub login
-          command: |
-            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USER" --password-stdin
+      - setup-aws-ecr
```

Comment:
This populates AWS_ECR_URL?

Reply:
no, that action logs into ecr for us, AWS_ECR_URL is an env var set within circleci